contracts/unique/ICollection.sol

```diff
@@ -7,13 +7,15 @@ import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 string constant ROYALTIES_PROPERTY = "royalties";
 
 interface ICollection is IERC721 {
-    function nextTokenId() external view returns (uint256);
+    function addCollectionAdmin(address newAdmin) external;
```

Comment:
Once more. I guess it would be good to add here also
collection_.addToCollectionAllowList(address(this));

To make it possible to mint tokens to the collection from the contract